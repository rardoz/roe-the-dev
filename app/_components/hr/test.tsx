import { render } from '@testing-library/react'
import HR from '.'

describe('HR', () => {
  it('should render', () => {
    expect(render(<HR />).container).toMatchSnapshot()
  })
})
