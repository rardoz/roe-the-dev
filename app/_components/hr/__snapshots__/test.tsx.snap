// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR should render 1`] = `
<div>
  <hr
    class="mt-5 mb-16 border-dotted border-spacing-8 border-8 border-t-0 border-purple-300"
  />
</div>
`;
